minimesosFile

```diff
@@ -0,0 +1,75 @@
+minimesos {
+    clusterName = "cook"
+    loggingLevel = "INFO"
+    mapAgentSandboxVolume = true
```

Comment:
Is there a reason to set this to true? It was causing problems on my linux box when I was trying to follow this.